### DISCUSSION TITLE
Supported/Maintained Public Singularity Image

### DISCUSSION THREAD

**Discussion by jjacobson95** [minio/minio#16441] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
I'm currently trying to create a web application that could be containerized in Singularity for use on a HPC. I've currently got a version for Docker that is working well and utilizes your public minio docker image. Getting this converted to work in singularity is taking more work.

**Describe the solution you'd like**
A public minio image for Singularity would be very useful for users that work with HPC.

**Describe alternatives you've considered**
You already have a well supported and functional docker image, but docker doesn't work on a HPC.

**Additional context**
It would be great if all the same features that exist in docker could be replicated for singularity.

**Comment by ravindk89** [comment, marked as answer]
Hi @jjacobson95 - can you give us a little more details on the errors you are seeing?

According to the [Singularity](https://docs.sylabs.io/guides/2.6/user-guide/singularity_and_docker.html) docs, Singularity should work with Docker images directly. It's better we understand what specific issue there is given Singularity's claim of Docker image compatibility.
